### PR TITLE
Linters-Config

### PR DESCRIPTION
**Pull request by davmba**
### My first Microverse project.
**In this project:**

- A repo was created
- A branch was created from the main
- linters files were added to the branch and 
- A pull request was created

### CODE REVIEW

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @davmba   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Good commit messages


## Required Changes ♻️

- [ ] Please try to Add descriptive  README file to your project. Kindly add your GitHub and LinkedIn username at the author section and fix the MIT License link in your readme file, the Link is broken if you don't have License you can remove it.

- [ ]  Kindly make sure your PR has a descriptive summary (details) describing introduced changes in more detail. To do that you can click on the three dots of the description box. see the screen shoot of the box on the screenshot below ⬇️ and click edit and you will be able to write.

Capture

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
```
Comment (HAMMAS-SALEEM) [inline .github/workflows/linters.yml:4]: ### Descriptive Summary Example :heavy_check_mark:
hammas-summary

- [ ] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Welcome to Microverse :tada: :tada: :bouquet: :bouquet:
# Required Changes :recycle:

almost-there

## Hey @davmba :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Following Linters
:heavy_check_mark: Following Some Requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @davmba,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality✅
📌 Good commit messages ✅
📌Professional README ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**